src/System.Data.Odbc/tests/System.Data.Odbc.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{F3E72F35-0351-4D67-2209-725BCAD807BA}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetsWindows)' == 'true'">$(DefineConstants);TARGETSWINDOWS</DefineConstants>
```

Comment:
Many other projects are using `TargetsWindows` for the define, and I think it is also what the coding conventions tells you to use.